ckanext/validation/webassets/js/module-modal-dialog.js

```diff
@@ -15,6 +17,7 @@ this.ckan.module('modal-dialog', function (jQuery) {
      */
     initialize: function () {
       jQuery.proxyAll(this, /_on/);
+      console.log('HERE', this)
```

Comment:
What is the purpose of this log?

Reply:
thanks for pointing it, I already removed it. It was there to debug why the report json was working in the validation report template but it wasn't in the modal dialog.